src/services/homework/hooks/index.js

```diff
@@ -94,127 +110,185 @@ const hasViewPermissionAfter = (hook) => {
 	} else {
 		// check if it is a single homework AND user has view permission
 		if (data.schoolId != undefined && !hasPermission(data)) {
-			return Promise.reject(new errors.Forbidden("You don't have permissions!"));
+			return Promise.reject(
+				new errors.Forbidden("You don't have permissions!")
+			);
 		}
 	}
-	(hook.result.data) ? (hook.result.data = data) : (hook.result = data);
-	(hook.result.data) ? (hook.result.total = data.length) : (hook.total = data.length);
+	hook.result.data ? (hook.result.data = data) : (hook.result = data);
+	hook.result.data
```

Comment:
Codacy found an issue: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)